sdk/storage/Azure.Storage.DataMovement/src/Azure.Storage.DataMovement.csproj

```diff
@@ -19,6 +19,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="System.Threading.Channels" />
+    <PackageReference Include="Azure.Core" />
```

Comment:
We should be able to remove this once Azure.Storage.Common releases with Azure.Core 1.34+

Reply:
Tracking: https://github.com/Azure/azure-sdk-for-net/issues/37630